Corona-Warn-App/src/test/java/de/rki/coronawarnapp/ui/submission/viewmodel/SubmissionOutOfGermanyViewModelTest.kt

```diff
@@ -0,0 +1,58 @@
+package de.rki.coronawarnapp.ui.submission.viewmodel
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import org.junit.Rule
+import org.junit.Test
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.Assertions.assertTrue
+
+class SubmissionOutOfGermanyViewModelTest {
+
+    private var submissionOutOfGermanyViewModel: SubmissionOutOfGermanyViewModel =
+        SubmissionOutOfGermanyViewModel()
+
+    // @JvmField could be removed when using @get:Rule
+    @Rule
+    @JvmField
+    var instantTaskExecutorRule: InstantTaskExecutorRule = InstantTaskExecutorRule()
+
+    @Test
+    fun textButtonClickedNone() {
+        submissionOutOfGermanyViewModel.buttonClicked(ButtonClicked.NONE)
+
+        submissionOutOfGermanyViewModel.yesAnswerActive.value?.let { assertFalse(it) }
```

Comment:
`?` to `!!` so tests fail on null.

Same for `?` below :point_down:

Reply:
I wanted to change it to !! but I discovered a bigger issue behind regarding transformations which took me some brain. I rewrote it and its now safe and will fail on null.